### ISSUE TITLE
Windows 11 - only finds partial libraries, if any?

### ISSUE THREAD

**Issue by Parrallathan** [PhilipK/BoilR#207]
Hey there,

I'm completely new to BoilR, so please forgive me if I'm making some obvious mistake, but: BoilR isn't finding all of the games in my various non-steam libraries. According to it, I've got no GOG or UPlay games (I do), and only a few games from EGS, Origin and Amazon Gaming. At first I thought it was only finding installed games, but that turned out not to be the case.

I made sure to find config.json for GOG after I saw in the command prompt that was it's issue - other than that it seems to be giving off what look like successful attempt messages?

Picture below in case it helps
![boilr1](https://user-images.githubusercontent.com/111685364/185788311-26224e11-3ab5-46c8-bbd9-804b22ae2953.jpg)
.

**Comment by PhilipK** [comment]
BoilR will only find the installed games, is that your problem with the Epic games?
Aoubthe Gog, what if you reset (clear the path setting), will it find the games?

**Comment by Parrallathan** [comment]
It only finds installed games? Very strange - I don't have any Origin games installed, but it found three of them.

EGS seems to be fine, it's only found my installed games. GOG doesn't want to find anything despite having a few installed - clearing the path setting doesn't seem to make it want to play ball.

**Comment by PhilipK** [comment]
Some times it find origin games that you purchased through steam, like mass
effect or dragon age for me

**Comment by Parrallathan** [comment]
I don't have these games on Steam, I'm afraid - and I didn't purchase them through Steam, either.

In case it's relevant, the culprits are Dead Space 2, Jade Empire Special Edition & Dragon Age 2.

**Comment by PhilipK** [comment]
So they are totally random? You never owned them before? Or no other user on the computer owns them?
If they show up, it is because origin has written in its config that they are installed, so I am wondering why that is.

**Comment by Parrallathan** [comment]
Seemingly random, yeah - Dead Space 2 was a freebie giveaway (Prime code for Origin) and the other two were physical CD keys redeemed on Origin. No idea what the trick there is.

**Comment by PhilipK** [comment]
So you own them, they are just not installed?

**Comment by Parrallathan** [comment]
That's correct, yeah - I own them, not through Steam, but they're not installed. That's why I thought BoilR did uninstalled games, too.

**Comment by PhilipK** [comment]
Have you had those games installed in the past?
I am wondering if Origin for some reason stlll thinks they are installed.

**Comment by Parrallathan** [comment]
I haven't had these games installed on this system, no. New rig I got December 2020, haven't played any of those since then.

Still can't find GOG games.
